src/main/kotlin/org/team5419/fault/math/pid/PIDF.kt

```diff
@@ -59,16 +69,45 @@ public class PIDF {
                 accumulator += value
             }
         }
+    }
+
+    public fun calculate(): Double {
+        var total = 0.0
+
+        if (kP != 0.0) {
+            total += kP * error
+        }
+
```

Comment:
Why add the if statements?

Reply:
Admittedly, that if statement is basically useless.